docs/ref/templates/builtins.txt

```diff
@@ -515,15 +516,6 @@ of how ``x in y`` will be interpreted::
 
 Not contained within. This is the negation of the ``in`` operator.
 
-The comparison operators cannot be 'chained' like in Python or in mathematical
```

Comment:
I've moved it above the list of operators as suggested, and made the change as a separate commit.